docs/input/docs/ci-systems/travisci.md

```diff
@@ -0,0 +1,7 @@
+---
+Order: 50
+Title: TravisCI
+Description: Running on TravisCI
+---
+
+This documentation is currently missing. 
```

Comment:
@gep13 , @AdmiringWorm I can't seem to get a really simple build working in travis. See https://travis-ci.org/github/nils-a/TestCakeRecipeBuildSystems/builds/726660450. Do we have some examples on this?

Reply:
Does this help: https://github.com/cake-contrib/Cake.Recipe.Tests/blob/develop/.travis.yml  This is how @AdmiringWorm is running a Travis Build for the integration tests for Cake.Recipe.

Reply:
Looking at your build, I see you only have .NET Core 2.1.
If you are using `dotnet tool restore` you need the 3.1, as restoring tools that way isn't available for 2.1.

Reply:
@AdmiringWorm Thanks. Seems I have much to learn regarding .NET Core.

Reply:
I have added a running example for building on travis.